README.md

```diff
@@ -88,7 +87,30 @@ asreview semantic_clustering -f van_de_Schoot_2017.csv -o output.csv
 
 If an output file is not specified, `output.csv` is used as output file name.
 
-### Dashboard
+### Test file
+```shell
+asreview semantic_clustering -t -o <output_file.csv>
+```
+
+Using `-t` instead of `-f` uses the
+[`van_de_Schoot_2017`](https://asreview.readthedocs.io/en/latest/intro/datasets.html#featured-datasets)
+dataset as input file. This way, the plugin can easily be tested.
```

Comment:
Just a side note: https://asreview.readthedocs.io/en/latest/API/generated/asreview.data.load_data.html has also support for the benchmark datasets. So you can have for example: `asreview semantic_clustering -f benchmark:van_de_Schoot_2017`. This is more in line with ASReview simulate interface. 

I think this can replace the `-t` flag and simplify your codebase.

Reply:
Ha, didn't know that! Yes, that will simplify things a lot, removing the need for an output argument